lib/src/Base/Func/CMakeLists.txt

```diff
@@ -221,9 +223,9 @@ ot_install_header_file (InverseBoxCoxEvaluation.hxx)
 ot_install_header_file (InverseBoxCoxGradient.hxx)
 ot_install_header_file (InverseBoxCoxHessian.hxx)
 ot_install_header_file (InverseBoxCoxTransform.hxx)
-ot_install_header_file (TrendEvaluation.hxx)
+#ot_install_header_file (TrendEvaluation.hxx)
```

Comment:
Why is this line commented out?  Also 2 lines below.

Reply:
@dbarbier Fixed